config/routes.rb

```diff
@@ -467,7 +467,7 @@
       get 'transfert' => 'procedures#transfert', as: :transfert
       get 'close' => 'procedures#close', as: :close
       post 'transfer' => 'procedures#transfer', as: :transfer
-      resources :mail_templates, only: [:edit, :update]
+      resources :mail_templates, only: [:edit, :update, :show]
```

Comment:
```suggestion
      resources :mail_templates, only: [:edit, :update]
```
je ne vois pas ou cette route est utilisé ?